### PR TITLE
Implement metrics

### PR DESCRIPTION
**Pull request by dzove855**
:sparkles: Add Metrics path [#5 ]

### CODE REVIEW

**Review by lvenier — APPROVED** [review]
if you want to tag to a new release you must change the package.json with the new version

**Comment by dzove855** [comment]
Updated to 0.4.1

File: flows.json [hunk 1]
```diff
@@ -4695,5 +4703,293 @@
...
+        "format": "handlebars",
+        "syntax": "mustache",
+        "template": "# HELP netbox_docker_agent_container_running Show if a container is running\n# TYPE netbox_docker_agent_container_running gauge\n{{#container.running}}\nnetbox_docker_agent_container_running{name=\"{{Names.0}}\", state=\"{{ State }}\", status=\"{{ Status }}\"} 1\n{{/container.running}}\n\n# HELP netbox_docker_agent_container_exited Show if a container is exited\n# TYPE netbox_docker_agent_container_exited gauge\n{{#container.exited}}\nnetbox_docker_agent_container_exited{name=\"{{Names.0}}\", state=\"{{ State }}\", status=\"{{ Status }}\"} 1\n{{/container.exited}}\n\n# HELP netbox_docker_agent_container_stopped Show if a container is exited\n# TYPE netbox_docker_agent_container_stopped gauge\n{{#container.stopped}}\nnetbox_docker_agent_container_stopped{name=\"{{Names.0}}\", state=\"{{ State }}\", status=\"{{ Status }}\"} 1\n{{/container.stopped}}\n\n# HELP netbox_docker_agent_docker_daemon Show if the connection to the daemon is working\n# TYPE netbox_docker_agent_docker_daemon gauge\nnetbox_docker_agent_docker_daemon{socket=\"/var/run/docker.socket\"} {{container.docker_daemon}}",
```
Comment (fanshan) [inline flows.json:4800]: You need to add triple brace in order to remove HTML escaping by mustache (eg: `{{{ Status }}}`)

**Review by fanshan — CHANGES_REQUESTED** [review]

File: flows.json (hunk 1 above)
Comment (dzove855) [reply flows.json]: it's fixed!

**Review by fanshan — APPROVED** [review]

**Pull request merged**